### ISSUE TITLE
Rankcheck crashes when mouse is unplugged

### ISSUE THREAD

**Issue by thetimmorland** [Marukyu/RankCheck#11]

**Comment by Blatoy** [comment]
Windows 10 64bits, but I didn't encounter this issue recently and I don't have a stack trace, sorry.

**Comment by Blatoy** [comment]
Just tested with the mouse multiple times, it didn't crash. I'm using rankcheck v1.5.1

**Comment by thetimmorland** [comment]
unsure of how to provide a stack trace, but I was able to reproduce the error just now on 1.5.1. Awesomenauts had to be running for rankcheck to crash.

**Comment by frenchDolphin** [comment]
Here's the file since it was too big to upload: [https://expirebox.com/download/182de94b0269f9f826eb10eae8131b73.html](url)

**Comment by frenchDolphin** [comment]
For some reason clicking on the link doesn't work, but I think if you copy and paste it, it works fine. If not, I'll re-upload it as soon as I can.

**Comment by frenchDolphin** [comment]
Yep, sorry I took so long. I didn't realize that you could open it with Visual Studio and get the dump from there. Here: https://pastebin.com/rNsKNrJk

**Comment by frenchDolphin** [comment]
I had to use Google Drive because the file was too big. (That's why I tried to use that other service the other time). https://drive.google.com/file/d/101gQ4OkLLV-4_53Nrn5ywSK7P67lBhl-/view?usp=sharing
